runelite-client/src/main/java/net/runelite/client/ui/ClientUI.java

```diff
@@ -513,8 +513,18 @@ public void requestFocus()
 			OSXUtil.requestFocus();
 		}
 
-		frame.requestFocus();
-		giveClientFocus();
+		if (System.getProperty("os.name").equals("Windows 10"))
```

Comment:
We have OSType.getOSType() what is literally 2 lines above and I would prefer for you to use that, also use either switch or elseif